common/requirements-common.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/eclipse-velocitas/vehicle-model-generator.git@v0.3.0
```

Comment:
License header needed?

Reply:
To stay consistent with the other requirements.txts we are maintaining, let's not add a license header to this file